docs/guides/_toc.json

```diff
@@ -302,7 +302,7 @@
               "title": "Configure runtime options",
```

Comment:
Should this be "configure primitive options"? Since it's under the "primitives" topic 🤔

Reply:
> Should this be "configure primitive options"? Since it's under the "primitives" topic 🤔

Hmm - but we do at least mention other primitives (not just runtime) in the other section, and this is specifically about runtime options.